app/src/main/kotlin/com/wire/android/ui/home/conversations/details/editguestaccess/EditGuestAccessViewModel.kt

```diff
@@ -143,19 +144,28 @@ class EditGuestAccessViewModel @Inject constructor(
     fun updateGuestAccess(shouldEnableGuestAccess: Boolean) {
         updateState(editGuestAccessState.copy(isUpdatingGuestAccess = true, isGuestAccessAllowed = shouldEnableGuestAccess))
         when (shouldEnableGuestAccess) {
-            true -> updateGuestRemoteRequest(shouldEnableGuestAccess)
+            true -> updateGuestRemoteRequest(true)
```

Comment:
Is this a leftover from testing or typo? why do we want to force to always allow guest access?

Reply:
it's checking before if the values true, so the value passed will be always true. It was a warning actually